rllib/algorithms/algorithm_config.py

```diff
@@ -1377,6 +1376,9 @@ def environment(
                 (gym.wrappers.EnvCompatibility). If False, RLlib will produce a
                 descriptive error on which steps to perform to upgrade to gymnasium
                 (or to switch this flag to True).
+             action_mask_key: If observation is a dictionary, expect the value by
```

Comment:
Awesome, thanks for adding this so quickly! I think it's ready to merge now. Just waiting for tests to finish ..